### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by ibr5500**
### in this project I have:
- Update the local storage when checking the task as completed and uncompleted.
- Add a function that clears all completed tasks.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -43,11 +43,40 @@ tasks.addEventListener('keypress', (event) => {
...
   window.location.reload();
 });
 
+const clearAll = document.querySelector('#clear');
+clearAll.addEventListener('click', () => {
+  let tasks = [];
+    tasksList.map((task) => { // eslint-disable-line
+    if (!task.completed) {
+      tasks = [...tasks, task];
+    }
```
Comment (roshan-bajgain) [inline src/index.js]: - [ ] According to the project requirement Kindly implement a function for the "Clear all completed" button using the Array.filter(). 
Why you should implement Array.filter()?
   - Array.filter returns a new array based on the condition that you provide inside the filter method while array.map is used to loop through the list of items.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
- [x] Your live demo is not working in the README file. So kindly make your live-demo work.
- [x] Kindly update your README with the instruction to install and start the project while I try `npm start` but it doesn't work, it works when I try `npm run dev` so include it on README file.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Hi @roshan-bajgain,
I have made all the changes that you provided to me.
Thank you for the code review.

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Thank you, @elmar8287, for the code review.

**Pull request merged**